### ISSUE TITLE
2025-06-11, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#197]
# Today's Arxiv Papers

(1) [ALE-Bench: A Benchmark for Long-Horizon Objective-Driven Algorithm Engineering](http://arxiv.org/abs/2506.09050v1), 2025-06-10

(2) [VIKI-R: Coordinating Embodied Multi-Agent Cooperation via Reinforcement Learning](http://arxiv.org/abs/2506.09049v1), 2025-06-10

(3) [Understanding Task Vectors in In-Context Learning: Emergence, Functionality, and Limitations](http://arxiv.org/abs/2506.09048v1), 2025-06-10

(4) [Agentic Neural Networks: Self-Evolving Multi-Agent Systems via Textual Backpropagation](http://arxiv.org/abs/2506.09046v1), 2025-06-10

(5) [Autoregressive Semantic Visual Reconstruction Helps VLMs Understand Better](http://arxiv.org/abs/2506.09040v1), 2025-06-10

(6) [AbstentionBench: Reasoning LLMs Fail on Unanswerable Questions](http://arxiv.org/abs/2506.09038v1), 2025-06-10

(7) [FZOO: Fast Zeroth-Order Optimizer for Fine-Tuning Large Language Models towards Adam-Scale Speed](http://arxiv.org/abs/2506.09034v1), 2025-06-10

(8) [Router-R1: Teaching LLMs Multi-Round Routing and Aggregation via Reinforcement Learning](http://arxiv.org/abs/2506.09033v1), 2025-06-10

(9) [e3: Learning to Explore Enables Extrapolation of Test-Time Compute for LLMs](http://arxiv.org/abs/2506.09026v1), 2025-06-10

(10) [Comparing human and LLM proofreading in L2 writing: Impact on lexical and syntactic features](http://arxiv.org/abs/2506.09021v1), 2025-06-10

(11) [SPEED-RL: Faster Training of Reasoning Models via Online Curriculum Learning](http://arxiv.org/abs/2506.09016v1), 2025-06-10

(12) [Learning to Reason Across Parallel Samples for LLM Reasoning](http://arxiv.org/abs/2506.09014v1), 2025-06-10

(13) [Boosting Rust Unit Test Coverage through Hybrid Program Analysis and Large Language Models](http://arxiv.org/abs/2506.09002v1), 2025-06-10

(14) [SwS: Self-aware Weakness-driven Problem Synthesis in Reinforcement Learning for LLM Reasoning](http://arxiv.org/abs/2506.08989v1), 2025-06-10

(15) [AdaDec: Uncertainty-Guided Adaptive Decoding for LLM-based Code Generation](http://arxiv.org/abs/2506.08980v1), 2025-06-10

(16) [Propositional Logic for Probing Generalization in Neural Networks](http://arxiv.org/abs/2506.08978v1), 2025-06-10

(17) [Atomic-to-Compositional Generalization for Mobile Agents with A New Benchmark and Scheduling System](http://arxiv.org/abs/2506.08972v1), 2025-06-10

(18) [ADAM: Autonomous Discovery and Annotation Model using LLMs for Context-Aware Annotations](http://arxiv.org/abs/2506.08968v1), 2025-06-10

(19) [Step-Audio-AQAA: a Fully End-to-End Expressive Large Audio Language Model](http://arxiv.org/abs/2506.08967v1), 2025-06-10

(20) [WIP: Large Language Model-Enhanced Smart Tutor for Undergraduate Circuit Analysis](http://arxiv.org/abs/2506.08962v1), 2025-06-10

(21) [Can LLMs Ground when they (Don't) Know: A Study on Direct and Loaded Political Questions](http://arxiv.org/abs/2506.08952v1), 2025-06-10

(22) [FaithfulRAG: Fact-Level Conflict Modeling for Context-Faithful Retrieval-Augmented Generation](http://arxiv.org/abs/2506.08938v1), 2025-06-10

(23) [BioLangFusion: Multimodal Fusion of DNA, mRNA, and Protein Language Models](http://arxiv.org/abs/2506.08936v1), 2025-06-10

(24) [Can A Gamer Train A Mathematical Reasoning Model?](http://arxiv.org/abs/2506.08935v1), 2025-06-10

(25) [What Limits Virtual Agent Application? OmniBench: A Scalable Multi-Dimensional Benchmark for Essential Virtual Agent Capabilities](http://arxiv.org/abs/2506.08933v1), 2025-06-10

(26) [PropMEND: Hypernetworks for Knowledge Propagation in LLMs](http://arxiv.org/abs/2506.08920v1), 2025-06-10

(27) [Quantifying Mix Network Privacy Erosion with Generative Models](http://arxiv.org/abs/2506.08918v1), 2025-06-10

(28) [Dialect Normalization using Large Language Models and Morphological Rules](http://arxiv.org/abs/2506.08907v1), 2025-06-10

(29) [From Legal Texts to Defeasible Deontic Logic via LLMs: A Study in Automated Semantic Analysis](http://arxiv.org/abs/2506.08899v1), 2025-06-10

(30) [AdversariaL attacK sAfety aLIgnment(ALKALI): Safeguarding LLMs through GRACE: Geometric Representation-Aware Contrastive Enhancement- Introducing Adversarial Vulnerability Quality Index (AVQI)](http://arxiv.org/abs/2506.08885v1), 2025-06-10

(31) [Your Brain on ChatGPT: Accumulation of Cognitive Debt when Using an AI Assistant for Essay Writing Task](http://arxiv.org/abs/2506.08872v1), 2025-06-10

(32) [Adapting Vision-Language Foundation Model for Next Generation Medical Ultrasound Image Analysis](http://arxiv.org/abs/2506.08849v1), 2025-06-10

(33) [Design Patterns for Securing LLM Agents against Prompt Injections](http://arxiv.org/abs/2506.08837v1), 2025-06-10

(34) [The impact of fine tuning in LLaMA on hallucinations for named entity extraction in legal documentation](http://arxiv.org/abs/2506.08827v1), 2025-06-10

(35) [Measuring Data Science Automation: A Survey of Evaluation Tools for AI Assistants and Agents](http://arxiv.org/abs/2506.08800v1), 2025-06-10

(36) [Paths to Causality: Finding Informative Subgraphs Within Knowledge Graphs for Knowledge-Based Causal Discovery](http://arxiv.org/abs/2506.08771v1), 2025-06-10

(37) [AraReasoner: Evaluating Reasoning-Based LLMs for Arabic NLP](http://arxiv.org/abs/2506.08768v1), 2025-06-10

(38) [EDINET-Bench: Evaluating LLMs on Complex Financial Tasks using Japanese Financial Statements](http://arxiv.org/abs/2506.08762v1), 2025-06-10

(39) [Enhancing Accuracy and Maintainability in Nuclear Plant Data Retrieval: A Function-Calling LLM Approach Over NL-to-SQL](http://arxiv.org/abs/2506.08757v1), 2025-06-10

(40) [Factors affecting the in-context learning abilities of LLMs for dialogue state tracking](http://arxiv.org/abs/2506.08753v1), 2025-06-10

(41) [Unlocking the Potential of Large Language Models in the Nuclear Industry with Synthetic Data](http://arxiv.org/abs/2506.08750v1), 2025-06-10

(42) [Towards Secure and Private Language Models for Nuclear Power Plants](http://arxiv.org/abs/2506.08746v1), 2025-06-10

(43) [Consistent Paths Lead to Truth: Self-Rewarding Reinforcement Learning for LLM Reasoning](http://arxiv.org/abs/2506.08745v1), 2025-06-10

(44) [Breaking the ICE: Exploring promises and challenges of benchmarks for Inference Carbon & Energy estimation for LLMs](http://arxiv.org/abs/2506.08727v1), 2025-06-10

(45) [Improved LLM Agents for Financial Document Question Answering](http://arxiv.org/abs/2506.08726v1), 2025-06-10

(46) [Explainable Compliance Detection with Multi-Hop Natural Language Inference on Assurance Case Structure](http://arxiv.org/abs/2506.08713v1), 2025-06-10

(47) [ConfPO: Exploiting Policy Model Confidence for Critical Token Selection in Large Language Model Preference Optimization](http://arxiv.org/abs/2506.08712v1), 2025-06-10

(48) [Educators' Perceptions of Large Language Models as Tutors: Comparing Human and AI Tutors in a Blind Text-only Setting](http://arxiv.org/abs/2506.08702v1), 2025-06-10

(49) [On the Ethics of Using LLMs for Offensive Security](http://arxiv.org/abs/2506.08693v1), 2025-06-10

(50) [Brevity is the soul of sustainability: Characterizing LLM response lengths](http://arxiv.org/abs/2506.08686v1), 2025-06-10

(51) [Mitigating Reward Over-optimization in Direct Alignment Algorithms with Importance Sampling](http://arxiv.org/abs/2506.08681v1), 2025-06-10

(52) [Enhancing Reasoning Capabilities of Small Language Models with Blueprints and Prompt Template Search](http://arxiv.org/abs/2506.08669v1), 2025-06-10

(53) [Summarization for Generative Relation Extraction in the Microbiome Domain](http://arxiv.org/abs/2506.08647v1), 2025-06-10

(54) [TableDreamer: Progressive and Weakness-guided Data Synthesis from Scratch for Table Instruction Tuning](http://arxiv.org/abs/2506.08646v1), 2025-06-10

(55) [MEMETRON: Metaheuristic Mechanisms for Test-time Response Optimization of Large Language Models](http://arxiv.org/abs/2506.08643v1), 2025-06-10

(56) [Approaching Dialogue State Tracking via Aligning Speech Encoders and LLMs](http://arxiv.org/abs/2506.08633v1), 2025-06-10

(57) [Leveraging LLMs to Evaluate Usefulness of Document](http://arxiv.org/abs/2506.08626v1), 2025-06-10

(58) [RAISE: Enhancing Scientific Reasoning in LLMs via Step-by-Step Retrieval](http://arxiv.org/abs/2506.08625v1), 2025-06-10

(59) [Generalizing while preserving monotonicity in comparison-based preference learning models](http://arxiv.org/abs/2506.08616v1), 2025-06-10

(60) [Sample Efficient Demonstration Selection for In-Context Learning](http://arxiv.org/abs/2506.08607v1), 2025-06-10

(61) [RE-oriented Model Development with LLM Support and Deduction-based Verification](http://arxiv.org/abs/2506.08606v1), 2025-06-10

(62) [Hateful Person or Hateful Model? Investigating the Role of Personas in Hate Speech Detection by Large Language Models](http://arxiv.org/abs/2506.08593v1), 2025-06-10

(63) [CounselBench: A Large-Scale Expert Evaluation and Adversarial Benchmark of Large Language Models in Mental Health Counseling](http://arxiv.org/abs/2506.08584v1), 2025-06-10

(64) [The Geometries of Truth Are Orthogonal Across Tasks](http://arxiv.org/abs/2506.08572v1), 2025-06-10

(65) [Detecting State Manipulation Vulnerabilities in Smart Contracts Using LLM and Static Analysis](http://arxiv.org/abs/2506.08561v1), 2025-06-10

(66) [From Pixels to Graphs: using Scene and Knowledge Graphs for HD-EPIC VQA Challenge](http://arxiv.org/abs/2506.08553v1), 2025-06-10

(67) [Efficient Post-Training Refinement of Latent Reasoning in Large Language Models](http://arxiv.org/abs/2506.08552v1), 2025-06-10

(68) [DeepForm: Reasoning Large Language Model for Communication System Formulation](http://arxiv.org/abs/2506.08551v1), 2025-06-10

(69) [Safe and Economical UAV Trajectory Planning in Low-Altitude Airspace: A Hybrid DRL-LLM Approach with Compliance Awareness](http://arxiv.org/abs/2506.08532v1), 2025-06-10

(70) [Teaching Physical Awareness to LLMs through Sounds](http://arxiv.org/abs/2506.08524v1), 2025-06-10

(71) [MLVTG: Mamba-Based Feature Alignment and LLM-Driven Purification for Multi-Modal Video Temporal Grounding](http://arxiv.org/abs/2506.08512v1), 2025-06-10

(72) [The Predictive Brain: Neural Correlates of Word Expectancy Align with Large Language Model Prediction Probabilities](http://arxiv.org/abs/2506.08511v1), 2025-06-10

(73) [MasHost Builds It All: Autonomous Multi-Agent System Directed by Reinforcement Learning](http://arxiv.org/abs/2506.08507v1), 2025-06-10

(74) [DRAGged into Conflicts: Detecting and Addressing Conflicting Sources in Search-Augmented LLMs](http://arxiv.org/abs/2506.08500v1), 2025-06-10

(75) [EtiCor++: Towards Understanding Etiquettical Bias in LLMs](http://arxiv.org/abs/2506.08488v1), 2025-06-10

(76) [Fairness is Not Silence: Unmasking Vacuous Neutrality in Small Language Models](http://arxiv.org/abs/2506.08487v1), 2025-06-10

(77) [RHealthTwin: Towards Responsible and Multimodal Digital Twins for Personalized Well-being](http://arxiv.org/abs/2506.08486v1), 2025-06-10

(78) [Efficient Fireworks Algorithm Equipped with an Explosion Mechanism based on Student's T-distribution](http://arxiv.org/abs/2506.08484v1), 2025-06-10

(79) [Efficient Context Selection for Long-Context QA: No Tuning, No Iteration, Just Adaptive-$k$](http://arxiv.org/abs/2506.08479v1), 2025-06-10

(80) [Detecting Harmful Memes with Decoupled Understanding and Guided CoT Reasoning](http://arxiv.org/abs/2506.08477v1), 2025-06-10

(81) [AsFT: Anchoring Safety During LLM Fine-Tuning Within Narrow Safety Basin](http://arxiv.org/abs/2506.08473v1), 2025-06-10

(82) [A Survey on Large Language Models for Mathematical Reasoning](http://arxiv.org/abs/2506.08446v1), 2025-06-10

(83) [SakugaFlow: A Stagewise Illustration Framework Emulating the Human Drawing Process and Providing Interactive Tutoring for Novice Drawing Skills](http://arxiv.org/abs/2506.08443v1), 2025-06-10

(84) [Olica: Efficient Structured Pruning of Large Language Models without Retraining](http://arxiv.org/abs/2506.08436v1), 2025-06-10

(85) [Low-resource domain adaptation while minimizing energy and hardware resource consumption](http://arxiv.org/abs/2506.08433v1), 2025-06-10

(86) [CAF-I: A Collaborative Multi-Agent Framework for Enhanced Irony Detection with Large Language Models](http://arxiv.org/abs/2506.08430v1), 2025-06-10

(87) [Better Reasoning with Less Data: Enhancing VLMs Through Unified Modality Scoring](http://arxiv.org/abs/2506.08429v1), 2025-06-10

(88) [Know-MRI: A Knowledge Mechanisms Revealer&Interpreter for Large Language Models](http://arxiv.org/abs/2506.08427v1), 2025-06-10

(89) [Transforming Expert Knowledge into Scalable Ontology via Large Language Models](http://arxiv.org/abs/2506.08422v1), 2025-06-10

(90) [Improved Scaling Laws in Linear Regression via Data Reuse](http://arxiv.org/abs/2506.08415v1), 2025-06-10

(91) [Large Language Models Have Intrinsic Meta-Cognition, but Need a Good Lens](http://arxiv.org/abs/2506.08410v1), 2025-06-10

(92) [TACTIC: Translation Agents with Cognitive-Theoretic Interactive Collaboration](http://arxiv.org/abs/2506.08403v1), 2025-06-10

(93) [mSTEB: Massively Multilingual Evaluation of LLMs on Speech and Text Tasks](http://arxiv.org/abs/2506.08400v1), 2025-06-10

(94) [Reinforce LLM Reasoning through Multi-Agent Reflection](http://arxiv.org/abs/2506.08379v1), 2025-06-10

(95) [EIFBENCH: Extremely Complex Instruction Following Benchmark for Large Language Models](http://arxiv.org/abs/2506.08375v1), 2025-06-10

(96) [Draft-based Approximate Inference for LLMs](http://arxiv.org/abs/2506.08373v1), 2025-06-10

(97) [Mitigating Posterior Salience Attenuation in Long-Context LLMs with Positional Contrastive Decoding](http://arxiv.org/abs/2506.08371v1), 2025-06-10

(98) [CC-RAG: Structured Multi-Hop Reasoning via Theme-Based Causal Graphs](http://arxiv.org/abs/2506.08364v1), 2025-06-10

(99) [DEAL: Disentangling Transformer Head Activations for LLM Steering](http://arxiv.org/abs/2506.08359v1), 2025-06-10

(100) [Reinforcement Fine-Tuning for Reasoning towards Multi-Step Multi-Source Search in Large Language Models](http://arxiv.org/abs/2506.08352v1), 2025-06-10
